pom.xml

```diff
@@ -116,6 +111,10 @@ THE SOFTWARE.
     <findbugs.effort>Max</findbugs.effort>
     <findbugs.threshold>High</findbugs.threshold>
     <findbugs.excludeFilterFile>${project.basedir}/../src/findbugs/findbugs-excludes.xml</findbugs.excludeFilterFile>
+
+    <!-- XXX keep these in sync -->
```

Comment:
```suggestion
    <!-- TODO keep these in sync -->
```

So why can these not be put in the BOM now?

Reply:
The checker is for a maven plugin not a regular dependency.

Hopefully they are not that tied together (the annotations are after all just annotations)